cordova-lib/src/platforms/platformsConfig.json

```diff
@@ -11,15 +9,12 @@
     "osx": {
         "hostos": ["darwin"],
         "parser_file": "../cordova/metadata/osx_parser",
-        "handler_file": "../plugman/platforms/ios",
```

Comment:
hmm, this might be an issue. OSX uses ios handler file. Maybe make a copy of `../plugman/platforms/ios` and rename it to `../plugman/platforms/osx` and update this file to use the copy